internal/provisioner/plain/pkg/git/git.go

```diff
@@ -0,0 +1,78 @@
+package git
+
+import (
+	"errors"
+	"fmt"
+	"net/url"
+	"strings"
+
+	"github.com/operator-framework/rukpak/api/v1alpha1"
+)
+
+const (
+	defaultDirectory = "./manifests"
```

Comment:
This seems like it should refer to another constant that exists somewhere in the code that defines the `plain+v1` spec. If that constant doesn't exist yet, perhaps we should add a TODO comment here?

Reply:
According to the spec (which we just merged) we will default to ./manifests for bundle git repos. So I propose leaving this as-is for now. Unless I'm misunderstanding what's being asked in which case we can revisit.